askama/src/lib.rs

```diff
@@ -163,7 +163,7 @@ impl fmt::Display for dyn DynTemplate {
 }
 
 #[cfg(test)]
-#[allow(clippy::blacklisted_name)]
+#[allow(clippy::disallowed_names)]
```

Comment:
Not strictly part of this PR, but maybe it would be good to comment which names exactly are disallowed?

Otherwise: LGTM